examples/anthos-service-mesh-multicluster/mysql-docker/app/ca.pem

```diff
@@ -0,0 +1,5 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Disclaimer: I'm no k8s expert but...

IIUC it's not a good practice to bake certificates into a docker image / container (which should only contain the bare minimum for the software of the app, and reading certs / config files from configmap or secret). 
Could this be done with a [letsencrypt](https://letsencrypt.org/) cert-manager instead?

Reply:
Nice catch. Fixed using a Kubernetes secret in the same yaml file where the image is pulled